sdk/spring/azure-spring-boot/src/samples/java/com/azure/spring/jms/QueueReceiveController.java

```diff
@@ -18,10 +18,10 @@
 
     private static final String QUEUE_NAME = "<ServiceBusQueueName>";
 
-    private final Logger LOGGER = LoggerFactory.getLogger(QueueReceiveController.class);
+    private final Logger logger = LoggerFactory.getLogger(QueueReceiveController.class);
```

Comment:
Why don't we make this static?

Reply:
There is a lot of code like this:
```
private final Logger logger
```
If we modify, maybe we should modify together, in another PR.

Reply:
Fair enough.